DFXR3Editor/Defs/DefActionID.xml

```diff
@@ -683,22 +815,22 @@
 				<field name="" dataType=""/>
 			</Fields2>
 		</ActionID>
-		<ActionID ID="609" name="LightSource(Color)">
+		<ActionID ID="609" name="Light Source">
```

Comment:
Removing Function Style syntax in the name is a dangerous choice, however it doesn't serve much purpose to anyone who's not familiar to it, i suggest giving action ID's visual cues that allow to easily identify them.